Tests/BluemixAppIDTests/WebAppPluginTest.swift

```diff
@@ -204,12 +204,12 @@ class WebAppPluginTest: XCTestCase {
                 super.init(response: response, routerStack: routerStack, request: request)
             }
             public override func redirect(_ path: String, status: HTTPStatusCode = .movedTemporarily)  -> RouterResponse {
-                if expectation == nil {
-                    XCTFail()
-                } else {
+                if let expectation = expectation {
+                    print("expectation: \(expectation)")
```

Comment:
Did you mean to leave this in?

Reply:
I did because their are about 10 tests which can fail here and without the expectation print out you can't tell which one of them actually failed. Since it is in the tests i thought the print statement would be ok.
I am happy to change it or just remove it if you have an alternative solution.

Reply:
I just don't like testsuites that make a lot of noise even when they're passing :) Not a big deal though.

Reply:
Ok I've removed the print line.